src/Operation/ListCollections.php

```diff
@@ -2,6 +2,8 @@
 
 namespace MongoDB\Operation;
 
+require 'src/functions.php';
+
```

Comment:
Composer should handle autoloading and requiring `src/functions.php`, so this shouldn't be necessary. Can you try regenerating your autoloaders again with `composer dump-autoload` or `composer update`?